packages/paste-core/primitives/box/src/types.ts

```diff
@@ -64,9 +64,7 @@ export interface BoxBaseStyleProps
   '-webkit-opacity'?: string;
 }
 
-export type BoxPseudoStyleProps = {
-  [key in keyof typeof PseudoPropStyles]?: BoxBaseStyleProps;
-};
+export type BoxPseudoStyleProps = Partial<Record<keyof typeof PseudoPropStyles, BoxBaseStyleProps>>;
```

Comment:
Note: updated this to use `Record` because it's slightly more robust.

```
// TS v4.6.2
type Record<K extends string | number | symbol, T> = { [P in K]: T; }
```

[More info found here](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type).